README.md

```diff
@@ -1,117 +1,108 @@
 # OpenZeppelin Contracts for Cairo
 
-[![Tests and linter](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml)
-[![codecov](https://codecov.io/github/OpenZeppelin/cairo-contracts/branch/main/graph/badge.svg?token=LFSZH8RPOL)](https://codecov.io/github/OpenZeppelin/cairo-contracts)
+[![Lint and test](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml)
 
-**A library for secure smart contract development** written in Cairo for [StarkNet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
+**A library for secure smart contract development** written in Cairo for [Starknet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
-## Usage
-
-> ## ⚠️ WARNING! ⚠️
->
+> **Warning**
 > This repo contains highly experimental code.
-> Expect rapid iteration.
+> It has no code coverage checks.
+> It hasn't been audited.
 > **Use at your own risk.**
 
-### First time?
+## Usage
 
-Before installing Cairo on your machine, you need to install `gmp`:
+> **Warning**
+> Expect rapid iteration.
+> Some contracts or features are not ready to be deployed.
+> Check the **Unsupported** section below.
 
-```bash
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-```
+### Prepare the environment
 
-> If you have any troubles installing gmp on your Apple M1 computer, [here’s a list of potential solutions](https://github.com/OpenZeppelin/nile/issues/22).
+1. [Install cairo](https://book.cairo-lang.org/ch01-01-installation.html)
```

Comment:
This guide already uses scarb as the way for installing cairo, I think it make sense to have just one link with something like: [Install Cairo and scarb](#).